grazer/src/main/java/edu/asu/diging/grazer/core/graphs/impl/GraphManager.java

```diff
@@ -29,11 +31,10 @@
 
 @Service
 public class GraphManager implements IGraphManager {
-
-    private final String PERSON_SIMPLE_TRIPLE = "person_simple_triple";
-    private final String PERSON_OBJECT_SIMPLE_TRIPLE = "person_object_simple_triple";
-    private final String PERSON_HAS_SOMEONE = "person_has_someone";
-    private final String SOMEONE_HAS_PERSON = "someone_has_person";
+    
+    private final String fileExtension = ".graphml";
+    private final String prefix = "PAT_";
+    private final String folderName = getClass().getResource("/transformations").getFile();
```

Comment:
constants should be uppercased; foldername should be the name of the folder ```/transformations```. you can get the path to the file in the init method.